### DISCUSSION TITLE
Extra link is not visible when using `TriggerDagRunOperator` inside a `task`

### DISCUSSION THREAD

**Discussion by r11234** [apache/airflow#43815] [Q&A]
I have the following setup:

```python
@task(task_id="example_task")
def example_task(arg1, arg2, **kwargs):
    TriggerDagRunOperator(
         task_id="..."
         trigger_dag_id="..."
         conf = { ... } # depends on arg1 and arg2
    ).execute(kwargs)
```

The issue I face is that I don't see the "Triggered DAG" extra link in the UI when I use example_task in a DAG. Is there a fix for this?

**Comment by potiuk** [comment]
Yes you cannot use "classic" operator in "task flow one". Use either one or the other. It literally makes no sense to wrap a classic task with taskflow. See the docs on how you can combine some tasks written in task flow way with some other tasks written in "classic way". There are examples.
